Functions/functions.js

```diff
@@ -0,0 +1,89 @@
+//  Task 1
+
+let mySum;
+let myDifference;
+let myMultiplication;
+let myDivision;
+
+function checking(value) {
+    if (value === 0 || value) {

```

Comment:
I know that you took this code from lesson, but now I think such checking will be better:

`typeof value === 'number' && !isNaN(value)`

Reply:
Yes, this is taken from the lesson. isNaN() from the Internet is a function that checks whether a value is not a number +"!" - on the contrary, numerical. Thank you very much.